### PR TITLE
[MRG] Add self version check

### LINKED ISSUE #36: Check for new conrad version
Compare user's `conrad.__version__` with the latest release on PyPI and suggest them to upgrade conrad periodically.

### PR DESCRIPTION
**Pull request by abhi-baireddy**
Fixes #36.

### CODE REVIEW

File: conrad/__main__.py [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
     cli()
-
+def get_current_version():
+    url = "https://pypi.org/pypi/conference-radar/json"
+    with urllib.request.urlopen(url) as response:
+        data = json.loads(response.read().decode())
+    return data["info"]["version"]
 
 if __name__ == "__main__":
+    latest_version = get_current_version()
```
Comment (vinayak-mehta) [inline conrad/__main__.py]: Can you put this in a function called `conrad_self_version_check`?

Comment (vinayak-mehta) [reply conrad/__main__.py]: Which we can call before `main()`.

File: conrad/__main__.py [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
 
 
 def main():
     from conrad.cli import cli
 
     cli()
-
+def get_current_version():
+    url = "https://pypi.org/pypi/conference-radar/json"
+    with urllib.request.urlopen(url) as response:
```
Comment (vinayak-mehta) [inline conrad/__main__.py]: Please use `requests`.

File: conrad/__main__.py [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
+        data = json.loads(response.read().decode())
+    return data["info"]["version"]
 
 if __name__ == "__main__":
+    latest_version = get_current_version()
+
+    if __version__ < latest_version:
```
Comment (vinayak-mehta) [inline conrad/__main__.py]: A tuple is being compared with a string here. You can instead use:

<pre>
>>> from setuptools.version import pkg_resources
>>> pkg_resources.parse_version("0.1.0") < pkg_resources.parse_version("0.1.1")
True<pre>

File: conrad/__main__.py [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
+        print("You are using conrad version " + __version__ + ", however, version " + latest_version + " is available.")
```
Comment (vinayak-mehta) [inline conrad/__main__.py]: Can you change it to a warning like it is done here: https://github.com/pypa/pip/blob/master/src/pip/_internal/self_outdated_check.py#L234

**Comment by abhi-baireddy** [comment]
I have made all the changes you requested. Please review them. Thanks for helping!

**Comment by vinayak-mehta** [comment]
Thanks for the self version check!

**Pull request merged**